AndroidProject/.gitignore

```diff
@@ -0,0 +1,14 @@
+*.iml
```

Comment:
We need only 1 .gitignore file. So this should be removed

Reply:
done